addons/mrp/static/src/components/bom_overview_line/mrp_bom_overview_line.xml

```diff
@@ -4,12 +4,12 @@
     <tr t-name="mrp.BomOverviewLine" owl="1">
         <td name="td_mrp_bom">
             <div t-attf-style="margin-left: {{ marginMultiplicator * 20 }}px">
-                <t t-if="data.level > 0 &amp;&amp; hasComponents">
-                    <button t-on-click="() => this.props.toggleFolded(identifier)" class="o_mrp_bom_unfoldable btn btn-light p-0" t-attf-aria-label="{{ props.isFolded ? 'Unfold' : 'Fold' }}" t-attf-title="{{ props.isFolded ? 'Unfold' : 'Fold' }}" style="margin-right: 1px">
-                        <i t-attf-class="fa fa-fw fa-caret-{{ props.isFolded ? 'right' : 'down' }}" role="img"/>
-                    </button>
-                </t>
-                <div t-attf-class="d-inline-block">
+                <div class="text-truncate">
```

Comment:
I think it's not needed to add a div in the div or use span). Just remove the existing ones. Also the kit icon should be correctly align also. (Due to an extra div). Also don't get why you added text-truncate
![Odoo-BoM-Overview (1)](https://github.com/odoo/odoo/assets/5512395/26b35980-677d-4493-8a6d-9d281b092610)